### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
I think it's almost done, but bootstrap hasn't been added
So I added Bootstrap

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -99,6 +100,12 @@
 * [Web Development Tutorials for Beginners in Hindi: HTML, CSS, JavaScript and more](https://www.youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg) - CodeWithHarry
 
 
+#### Bootstrap
+
+* [Bootstrap Tutorial In Hindi](https://youtu.be/vpAJ0s5S2t0) - CodeWithHarry
```
Comment (eshellman) [inline courses/free-courses-hi.md:105]: We do not permit shortened links. (youtube.com, not youtu.be)
Please read CONTRIBUTING.md guidelines!

**Comment by davorpa** [comment]
Duplicate of #7005

File: courses/free-courses-hi.md (hunk 1 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-hi.md:105]: that's my mistake

Comment (davorpa) [reply courses/free-courses-hi.md:105]: @GH05T-HUNTER5 could you add it again in #7005? Therefore we can avoid merge conflicts in TOC and Boostrap section

Comment (davorpa) [reply courses/free-courses-hi.md:105]: Add there the next line 106 too

**Comment by davorpa** [comment]
Conflicts are with Bootstrap section (added in both PRs but with different pair of resources) and changes made in TOC/sections fragment ids

**Comment by davorpa** [comment]
Conflicts are with Bootstrap section (added in both PRs but with different pair of resources) and changes made in TOC/sections fragment ids

**Comment by davorpa** [comment]
Duplicate of #7006

**Comment by davorpa** [comment]
Duplicate of #7006

File: courses/free-courses-hi.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-hi.md:105]: Resolved by commit 8b7ae419ae60a37a755c9a91ed406fc7c54b7030 part of PR #7006

**Pull request closed without merging**